test/functional/server/server_spec.lua

```diff
@@ -17,8 +17,7 @@ describe('serverstart(), serverstop()', function()
     eq('', eval('$NVIM_LISTEN_ADDRESS'))
   end)
 
-  it([[sets v:servername _only_ on nvim startup
-      (unless all servers are stopped)]], function()
+  it([[sets v:servername _only_ on nvim startup (unless all servers are stopped)]], function()
```

Comment:
My variant (additionally remove parenthesis, replace `[[]]` with `''` and move `function()` on the next line) keeps this line behind 80 characters cap.